core-bundle/src/Command/CrawlCommand.php

```diff
@@ -86,14 +87,27 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $io->title('Contao Crawler');
 
         $subscribers = $input->getOption('subscribers');
-        $queue = new InMemoryQueue();
         $baseUris = $this->escargotFactory->getCrawlUriCollection();
 
         if ($baseUris->containsHost('localhost')) {
             $io->warning('You are going to crawl localhost URIs. This is likely not desired and due to a missing domain configuration in your root page settings. You may also configure a fallback request context using "router.request_context.*" if you want to execute all CLI commands with the same request context.');
         }
 
         try {
+            switch ($input->getOption('queue')) {
```

Comment:
Did you move the `switch` statement inside the try-catch block on purpose?

Reply:
Not at all :D Fixedd.